AirCasting/CoreData/MeasurementStream.swift

```diff
@@ -9,6 +9,8 @@ enum MeasurementStreamSensorName: String {
     case pm1 = "AirBeam3-PM1"
     case pm2_5 = "AirBeam3-PM2.5"
     case pm10 = "AirBeam3-PM10"
```

Comment:
if we name mini, starting with `mini` shouldn't we be naming ab3 related sensors with ex. `ab3_pm10`?

Reply:
Yeah, makes sense 👌